### PR TITLE
paperless-ngx: 1.9.2 -> 1.10.2

### PR DESCRIPTION
**Pull request by babariviere**
###### Description of changes

Update paperless-ngx to 1.10.2.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/office/paperless-ngx/default.nix [hunk 1]
```diff
@@ -246,8 +227,18 @@ python.pkgs.pythonPackages.buildPythonApplication rec {
...
       --replace "--cov --cov-report=html" ""
+    # Fix test with typo
+    substituteInPlace src/paperless_tesseract/tests/test_parser.py \
+      --replace "this is awebp document" "this is a webp document"
   '';
 
+  disabledTests = [
+    # FileNotFoundError(2, 'No such file or directory'): /build/tmp...
+    "test_script_with_output"
+    # AssertionError: 10 != 4 (timezone/time issue)
```
Comment (gador) [inline pkgs/applications/office/paperless-ngx/default.nix:238]:
```suggestion
    # AssertionError: 10 != 4 (timezone/time issue)
    # Due to getting local time from modification date in test_consumer.py
```

File: pkgs/applications/office/paperless-ngx/default.nix [hunk 2]
```diff
@@ -246,8 +227,18 @@ python.pkgs.pythonPackages.buildPythonApplication rec {
     # Disable unneeded code coverage test
     substituteInPlace src/setup.cfg \
       --replace "--cov --cov-report=html" ""
+    # Fix test with typo
```
Comment (gador) [inline pkgs/applications/office/paperless-ngx/default.nix:230]:
```suggestion
    # OCR on nixos recognizes the space in the picture, upstreams CI doesn't. See https://github.com/paperless-ngx/paperless-ngx/pull/2216
```

**Review by gador — CHANGES_REQUESTED** [review]
LGTM

**Comment by erikarvstedt** [comment]
[Minor formatting fixup](https://github.com/erikarvstedt/nixpkgs/commits/paperless-ngx-1.10.2-ea). Fetch with:
```
git fetch https://github.com/erikarvstedt/nixpkgs paperless-ngx-1.10.2-ea:paperless-ngx-1.10.2-ea
```

**Comment by babariviere** [comment]
Applied, thanks.

**Comment by gador** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 206835` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>paperless-ngx</li>
<li>python310Packages.django-celery-results</li>
<li>python39Packages.django-celery-results</li>
</ul>
</details>

**Review by lukegb — APPROVED** [review]

File: pkgs/development/python-modules/django-celery-results/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    hash = "sha256-dapRlw21aRy/JCxqD/UMjN9BniZc0Om3cjNdBkNsS5k=";
+  };
+
+  propagatedBuildInputs = [
+    celery
+    django
+  ];
+
+  # Tests need access to a database.
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/django-celery-results/default.nix:27]: pythonImportsCheck is missing

**Pull request merged**